sdk/monitor/Azure.Monitor.Ingestion/README.md

```diff
@@ -0,0 +1,143 @@
+# Azure Monitor Ingestion client library for .NET
+
+This section should give out brief introduction of the client library.
+
+* First sentence: **Describe the service** briefly. You can usually use the first line of the service's docs landing page for this (Example: [Cosmos DB docs landing page](https://docs.microsoft.com/azure/cosmos-db/)).
+* Next, add a **bulleted list** of the **most common tasks** supported by the package or library, prefaced with "Use the client library for [Product Name] to:". Then, provide code snippets for these tasks in the [Examples](#examples) section later in the document. Keep the task list short but include those tasks most developers need to perform with your package.
```

Comment:
We should fill this out before the first release.  You can just copy from https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/monitor/azure-monitor-ingestion.